restapi/configure_weaviate.go

```diff
@@ -1226,13 +1226,15 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 		defer messaging.TimeTrack(time.Now())
 		messaging.DebugMessage("Starting GraphQL resolving")
 
+		errorResponse := &models.ErrorResponse{}
```

Comment:
Hmm, we define an `errorResponse`, and return in case of an error, but don't fill it with relevant information.

@moretea should fix this.

Reply:
Need to think about this a bit longer. Created https://github.com/creativesoftwarefdn/weaviate/issues/447